### PR TITLE
[MLDB-1342] set unnecessary boot message to debug and removed other m…

### PR DESCRIPTION
**Pull request by guyd**
…isleading messages

### CODE REVIEW

File: rest/http_rest_service.cc [hunk 1]
```diff
@@ -89,8 +89,7 @@ HttpRestConnection::
 sendErrorResponse(int responseCode, const Json::Value & error)
 {
     using namespace std;
-    cerr << "sent error response " << responseCode << " " << error
-         << endl;
+    cerr << "sent error response " << responseCode << " " << error << endl;
```
Comment (nicolaskruchten) [inline rest/http_rest_service.cc]: this is still cerr?

Comment (guyd) [inline rest/http_rest_service.cc]: I targeted messages that we shown at boot time.  This one can be done next time I touch that file!  :-)

**Comment by nicolaskruchten** [comment]
Looking good! Can we get similar treatment for the credentials daemon and the bit in the mldb_server that connects to it?

**Comment by guyd** [comment]
Yes.  I am on it.

**Comment by guyd** [comment]
looking into failed test

**Comment by guyd** [comment]
I've discussed, added an option to mute the output in credential daemon exec.

**Comment by nicolaskruchten** [comment]
+1

**Pull request merged**